docs/src/components/DocsBehaviorRoot.tsx

```diff
@@ -0,0 +1,72 @@
+import * as _ from 'lodash'
+import PropTypes from 'prop-types'
+import * as React from 'react'
+const behaviorMenuItems = require('docs/src/componentMenuBehaviors')
+import { Divider, Form, Grid, Menu, Segment, Visibility, SemanticCOLORS } from 'semantic-ui-react'
```

Comment:
Do we really need all these?  Don't we have stardust versions?

Reply:
I recognized that we need just Grid. Others imported components were leftovers. 
I took this part with the same styling from 'component' page. To have the same design. 
So if it necessary I can try use stardust grid.